### ISSUE TITLE
Installing Mothur

### ISSUE THREAD

**Issue by liluacrobat** [ubccr/software-layer#365]
Could you please install Mothur for 16S analysis (https://github.com/mothur/mothur/releases)?

**Comment by dsajdak** [comment]
Note to whoever works on this - there are very old [EB recipes](https://github.com/easybuilders/easybuild-easyconfigs/tree/develop/easybuild/easyconfigs/m/Mothur) which may work as a starting point

**Comment by tonykew** [comment]
Mothur version 1.48.1 has been built, and is going through internal validation,
I will update this request once Mothur has been published.

**Comment by tonykew** [comment]
Mothur version 1.48.1 has been published to the ccrsoft/2023.01 software release

```
login1$ module spider mothur
----------------------------------------------------------------------------
  mothur: mothur/1.48.1-Python-3.9.6
----------------------------------------------------------------------------
    Description:
      Mothur is a single piece of open-source, expandable software to fill
      the bioinformatics needs of the microbial ecology community.

    Properties:
      Bioinformatic libraries/apps

    You will need to load all module(s) on any one of the lines below before the "mothur/1.48.1-Python-3.9.6" module is available to load.

      gcc/11.2.0  openmpi/4.1.1
[...]
login1$ module load gcc/11.2.0 openmpi/4.1.1 ucx/1.13.1 mothur/1.48.1-Python-3.9.6
-------------------------------------------------------------------------------
The following dependent module(s) are not currently loaded: ucx/1.11.2 (required by: openmpi/4.1.1), zlib/1.2.11 (required by: pmix/4.1.0, openmpi/4.1.1)
-------------------------------------------------------------------------------

login1$ 
```

Note: I have explicitly loaded a newer version of the "ucx" module, and mother
loads a newer version of zlib so the above warnings can be ignored.

```
login1$ module -t list 2>&1 | grep -E 'ucx|zlib'
ucx/1.13.1
zlib/1.2.12
login1$ 
```


```
login1$ mothur --help
Linux version

Using ReadLine,Boost,HDF5
mothur v.1.48.1
Last updated: 5/15/24
by
Patrick D. Schloss

Department of Microbiology & Immunology

University of Michigan
http://www.mothur.org

When using, please cite:
Schloss, P.D., et al., Introducing mothur: Open-source, platform-independent, community-supported software for describing and comparing microbial communities. Appl Environ Microbiol, 2009. 75(23):7537-41.

Distributed under the GNU General Public License

Type 'help()' for information on the commands that are available

For questions and analysis support, please visit our forum at https://forum.mothur.org

Type 'quit()' to exit program

[NOTE]: Setting random seed to 19760620.

Script Mode



mothur > help()

NOTE: sens.spec assumes that only unique sequences were used to generate the distance matrix.
[...]
login1$ 
```
